client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts

```diff
@@ -4,23 +4,11 @@ const useStyles = makeStyles({
     airportDetails: {
         display: 'flex',
         flexDirection: 'row',
-        justifyContent: 'space-between',
         marginLeft: '-5.5vw',
         '@media screen and (max-width: 1280px)': {
             marginLeft: '-20.5vw'
         },
     },
-    airportCountryTextField: {
-        width: '10vw',
-        '@media screen and (min-width: 1450px)': {
```

Comment:
what about responsiveness if you deleted the media query?

Reply:
it wasnt used

Reply:
my bad will fix it